### DISCUSSION TITLE
have the issuer also support a list of widely known public providers

### DISCUSSION THREAD

**Discussion by acidjazz** [panva/openid-client#682] [Ideas]
This essentially: https://cs.opensource.google/go/x/oauth2/+/master:endpoints/endpoints.go

**Comment by panva** [comment]
It is very unlikely that this module would maintain static configurations for any Authorization Servers.

**Reply by acidjazz** [reply]
why not? they're extremely helpful.  they're required to use providers, which is why they're included in the official golang oauth2 package

**Reply by panva** [reply]
> they're required to use providers

They're not, FWIW the preferred way to get provider configurations is through discovery.

They're also not required to be part of the module. If you want to maintain a module that only exposes these configurations, you can.

**Reply by acidjazz** [reply]
looks like they're provided to me https://github.com/golang/oauth2/blob/master/google/google.go#L25-L30
